server/cli/dataPopulation/Dnd5eDataSeeder.ts

```diff
@@ -0,0 +1,101 @@
+import { v4 as uuidv4 } from 'uuid';
+
+import {
+  Dnd5eApiAdapter,
+  Dnd5eApiClient,
+  Dnd5dApiNormalizer
+} from '../dataPopulation/dnd5eapi';
+import {
+  IThirdPartyDndAdapter,
+  Monster,
+  SourceKinds
+} from './types';
+
+import { MonsterTemplate } from './templates';
+
+import { Creature } from '../../sequelize/db';
+
+// This should likely be moved up a level. There is a need to start sharing types between client and server
+import {
```

Comment:
This and another number of 5e specific functions will be moved to a core/types and core/system/rules directory in a future task.